include/zephyr/net/wifi_mgmt.h

```diff
@@ -693,6 +706,15 @@ struct wifi_channel_info {
 	enum wifi_mgmt_op oper;
 };
 
+/** Wi-Fi AP configuration parameter*/
+struct wifi_ap_config {
+	enum wifi_ap_param type;
+	/** Parameter used for setting ap_max_inactivity duration */
+	uint32_t ap_max_inactivity;
```

Comment:
```suggestion
	uint32_t max_inactivity;
```

The comment to add `ap_` was for earlier generic structure, as the structure itself is for AP, no need for prefix.